packages/workbox-background-sync/test/browser/request-queue.js

```diff
@@ -37,7 +37,20 @@ describe('request-queue tests', () => {
     chai.assert.isObject(queue._config);
   });
 
-  it('queueName is corrent', () =>{
+  it('initialize should re-fill the queue', async () => {
```

Comment:
Yes, please add a test case for when `this._idbQDb.get()` resolves to a null value, as it will the first time the SW starts up.